drivers/wifi/eswifi/eswifi_socket_offload.c

```diff
@@ -30,6 +30,7 @@ LOG_MODULE_DECLARE(LOG_MODULE_NAME);
  */
 #define SD_TO_OBJ(sd) ((void *)(sd + 1))
 #define OBJ_TO_SD(obj) (((int)obj) - 1)
+#define ESWIFI_INIT_CONTEXT	INT_TO_POINTER(0x50CE)
```

Comment:
This only deteriorates the situation, at least previously there was a comment alongside this magic number: `/* Assign dummy context SOCkEt(50CE) */` (character case problems?), now it's the magic number all on its own.

Reply:
@pfalcon What's the best fix here? Makes sense to move the comment to top as well?

Reply:
> What's the best fix here? Makes sense to move the comment to top as well?

Yes, that's what I'd do.

Reply:
@pfalcon Thanks, moved it.